ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -81,6 +81,10 @@ tests:
   container:
     from: src
 - as: e2e-aws
+  steps:
```

Comment:
i think the e2e-aws and the e2e-aws-upgrade might need to be switched?

e2e-aws is point to upgrade and upgrade is pointing to e2e-aws (if my eyes are correct)

Reply:
Good catch, fixed that